presto-main/src/test/java/com/facebook/presto/operator/BenchmarkGroupByHash.java

```diff
@@ -216,6 +224,37 @@ public long baselineBigArray(BaselinePagesData data)
         return pages.build();
     }
 
+    private static List<Page> createVarcharPages(int positionCount, int groupCount, List<Type> types, boolean hashEnabled)
```

Comment:
This is a weird API. If you assume that every type in `types` is VARCHAR, why not just pass in a `int columns` instead?

The same applies for `createPages`. You probably want to rename that method as well.